main.nf

```diff
@@ -24,12 +24,13 @@ if (!params.localOutDir) {
 pairFiles_ch = Channel.fromFilePairs( params.inputDir+"/*{1,2}.fastq.gz", size: 2, checkIfExists: true )
 
 process ERValign {
-    // tag ${sample}
+    tag !{sample}
```

Comment:
tag "${sample}"

the !{} notation is only for the template

Reply:
fixing it right now